regression-test.bash

```diff
@@ -24,5 +24,25 @@ echo "5. Running batcher-with-malicious-sequencer.bash"
 ./batcher-with-malicious-sequencer.bash
 echo "Completed batcher-with-malicious-sequencer.bash"
 
+echo "6. Running batcher-e2e.bash"
```

Comment:
Can we not like automate this to look for all bash files in this folder and with name `test` and run them so that we dont have to manually add each file?